frontend/src/__tests__/cypress/cypress/utils/should.ts

```diff
@@ -38,8 +38,19 @@ const form = {
   },
 };
 
+const sort = {
+  sortedByAscending: ($subject: JQuery<HTMLElement>) => {
```

Comment:
Personal preference here but I prefer a more concise name of `sortAscending` and `sortDescending`:
```suggestion
  sortAscending: ($subject: JQuery<HTMLElement>) => {
```